test/e2e/ibmcloud_test.go

```diff
@@ -92,6 +92,13 @@ func TestCreatePeerPodWithJob(t *testing.T) {
 	doTestCreatePeerPodWithJob(t, assert)
 }
 
+func TestCreatePeerPodWithUserImage(t *testing.T) {
```

Comment:
I think the name of this test case and the name of the `doTestCreatePeerPodWithUserImage` function should be different. I think the purpose of this test case it to test fetching logs from a pod container. The fact the pod container image is a user-built (or test case built) image is, I believe, circumstantial. So I think a name like `TestCreatePeerPodAndCheckLogs`.

Reply:
Hi @mattarnoatibm ,
There are test case which  need to be merged after this PR, which checks the logs of workdir, envvariable, etc
So can we keep this name as `TestCreatePeerPodAndCheckUserLogs`, `TestCreatePeerPodAndCheckWorkDirectoryLogs`...  ?

Reply:
Suggestion - since this test case checks the output of a container image entrypoint command, rename the test case to `TestCreatePeerPodAndCheckOutputOfEntrypointCmd`?

Reply:
So I'm not sure that this works fully like this Sudharshan? We have a number of tests that do this behaviour, but they will use different images and expect different logged output, so I think whilst we could have a basic test called `TestCreatePeerPodAndCheckOutputOfEntrypointCmd`, we would need to pass in the pod name, image, expected log, testcase name, description etc to have it unique for the 3/4 different test cases and at that point I'm not sure if it's messier to have a shared function with ~7/8 parameters rather than individual ones?

Reply:
I'm not a huge fan of using the map to get around there being a lot of parameters. I guess maybe the best thing to do it go back to the `TestCreatePeerPodAndCheckUserLogs` name and then look at refactoring it once we add more tests with common code, rather than attempting to prematurely refactor for the future?

Reply:
Hi @stevenhorsman ,
Reverted the test name to `TestCreatePeerPodAndCheckUserLogs`

Reply:
It still looks to be named `doTestCreatePeerPodAndCheckOutputOfEntrypointCmd`? Anyway, the main thing is to remove the map/extra parameters and review that once we add the env/workdir test